### ISSUE TITLE
#pixelscamp #forever

### ISSUE THREAD

**Issue by frm** [PixelsCamp/pixelscamp-forever#8]
I'm a proud member of the Pixels Camp community, I have read the [rules and requirements](https://github.com/PixelsCamp/pixelscamp-forever/blob/master/README.md), and I believe I qualify to get a free _Pixels Camp Forever!_ t-shirt.

This is my public proof on social media:

https://twitter.com/frmcodes/status/1305460547018727426

### Encrypted Name and Shipping Address:

```
U2FsdGVkX19BokVbzkA0Winevx9N22aHlERYWpypLAdrBf77qZngZgOyDvTgg/N28smRqHV57Kvd4q3QnTDDVw3zfdmiKRq3d/Wy2zngLnc=
```

### Encrypted Shared Key:

```
eUhrdcXXsGjacx7+6r9UNrtDNmJnrvOMMwz2FYoP1LoMQQXp+L4aVGTgsQ3BZJWUkLvimOxzYyHubMQgPFcFoPrcKVMa0imceEEfckdz5UnxPpSJ42+G4bbXzp2b8LEiBi54RN01NM+QLy8gikoaGz8Sxlaepot8grpYU3V/gaH6BukZ5hM6jOv5lfJEsbUY96tkrfc1CT1Gco/FTPVrWZqGgl6lLELy6+T5Xmp/pPkWzJvkgs+Ar8yI9WNdpWZb06hJibbCJpu/rO35kDIuK1R3xYzMCBMjb8qRaaPswhWQCkTSEbFz44wKlszIcVaZIgnZ0oVGNeUlF3z35KxRWQ==
```

**Comment by carlosefr** [comment]
Hi @frm 

Can you please regenerate both encrypted bits using the updated information in the README? The initial public key was wrong and has been replaced with the correct key, plus the OpenSSL commands were updated to ensure (some) compatibility between versions.

Sorry about the trouble.

**PS:** If you have a preference for t-shirt size, we'll try to honor it if we can.

**Comment by frm** [comment]
Just a PSA here: macOS comes with LibreSSL installed by default which doesn't allow for the `-pbkdf2` flag on the following command:

```bash
openssl aes-256-cbc -e -in file.txt -md sha256 -pbkdf2 -pass file:random.key | base64 > file.enc
```

macOS users have to install a different version:

```bash
/usr/local/opt/openssl@1.1/bin/openssl aes-256-cbc -e -in file.txt -md sha256 -pbkdf2 -pass file:random.key | base64 > file.enc
```

Or for a more permanent solution:

```bash
brew install openssl # or brew upgrade openssl@1.1
echo "export PATH='/usr/local/opt/openssl@1.1/bin:$PATH'" > ~/.bash_profile # or ~/.zprofile or whatever
```

**Comment by frm** [comment]
I'm a proud member of the Pixels Camp community, I have read the [rules and requirements](https://github.com/PixelsCamp/pixelscamp-forever/blob/master/README.md), and I believe I qualify to get a free _Pixels Camp Forever!_ t-shirt.

This is my public proof on social media:

https://twitter.com/frmcodes/status/1305460547018727426

### Encrypted Name and Shipping Address:

```
U2FsdGVkX1+xi4w0HEVA3uszx7+S1ZIp0unOVfPKxoZLqsBhYlIWceYxUbfcRARsdGHSsXH+/kOVvw/Kir6Gd5j/uQJyGuHVIbqtqcd3V9A=
```

### Encrypted Shared Key:

```
VCWk9eTV1KtlHEToQxGwoEodzYOO336RmptZCaUKccN6IiPvkzMvZibq2Yj26m3eQKt8k7DYGGggVZdb/pYuRqIfmDkaVxs6Bpeqdero5vhTsXQ/5uwy20raMj2RWv0kGcbceWudMdC3WY7kJM0bNumm69NTNwd9cAele3ASjBSLpDnMzuMq8sD/xU/IHdYnOLkWMIxIY3LbzQPOQZ8Bbg19G8X9Q1ULjz0dr1vaeQqvj5kes4/x2UBTg58biI1rYONADugExJxd0IxSO3K9Sbgm6WRyzJ0wVIBZ/BO62F9dSdedDCybay49uzD8EEFY2STPnfrD6gvaHndeRF+WOGHgzSS1BtFw3rN84bCB8KhRrGRsUXF0IE9kZx9DgOst5/bdJScBRCuNeaV4nrB0pcqJIhMsaY5V8cQ7ktywvm8r5lZAMfmv6gZ8InLDW1RbyF/63sYvrDutTmlTQBVipYPZ+iCzrUdlLaxXsYK13chZq1O643rEk8KwDJ42KZ/WpzjJEbuzC+efGxCYuHX9FSpQt3VqV2BsB/yYVCmkuvbdUqY5EleEkaRx7qPgW9Sbwc1YJtSheGnGLhB+/uhLjmZp68/lvmXVudIxi/RvPs0vvVTPwzq/fl95BVoFMpZkAGy4H29BTg5vBBZqqDIoDGxpdlbFvhMBseoFqxJxZ7c=
```

**Comment by carlosefr** [comment]
Yeah, in the README I mentioned that on macOS we tested with OpenSSL from homebrew.

**Comment by carlosefr** [comment]
Any preference for the t-shirt size?

**Comment by frm** [comment]
M, if possible

**Comment by carlosefr** [comment]
Hi @frm ,

The parcel carrier requires a phone number for the delivery. This needs to be done as quickly as possible, so you can just send it via email to org@pixels.camp. Please mention your GitHub username in the email, so we can match it.

Alternatively, you can send a direct message to me (carlosefr) on the Pixels Camp Slack.

**Comment by frm** [comment]
👍 done

**Comment by carlosefr** [comment]
Unfortunately, the parcel has been deemed undeliverable by the carrier (CTT) at the address provided and has been returned to the sender (the t-shirt manufacturer). It'll be kept safe and hopefully we can give it to you through other means in the future.

If you have any questions, please ask via DM on Slack.
